ccache.mk

```diff
@@ -53,11 +55,15 @@ endif
 ccache-package: ccache-stage
 	# ccache.mk Package Structure
 	rm -rf $(BUILD_DIST)/ccache
+	rm -rf $(BUILD_DIST)/ccache/usr/
+	mkdir -p $(BUILD_STAGE)/ccache/usr/libexec/ccache/
```

Comment:
This should be done after building, not during packaging. Wrong prefix too.

Reply:
I probably don't need line 58 at all. I will move line 59 to where I think it belongs. Give me a moment.

Reply:
Fixed on [latest commit](https://github.com/ProcursusTeam/Procursus/pull/569/commits/a0d7f6f14148df6f9cb52c6a7726ff7953ed055f)